src/airbrake.cr

```diff
@@ -49,22 +49,29 @@ module Airbrake
   class Config
     property :project_id
     property :project_key
+    property :host
+    property :port
+    property :secure
+
+    def uri
+      self.host   ||= "airbrake.io"
+      self.port   ||= 443
```

Comment:
I think in this case a little bit more docs is better than extra code. Speaking of the documentation, could you please document the `host` option?

With regard to the other two, I'd like to discuss this to find the best approach.